api/administration-sm/administration-sm-openapi.yaml

```diff
@@ -155,8 +155,8 @@ components:
       scheme: bearer
       bearerFormat: JWT
 info:
-  title: Console SM Admin API
-  description: Access the administration API of Console SM.
+  title: Administration API (Self-Managed)
+  description: Access the administration API of Console Self-Managed.
```

Comment:
@urbanisierung is there an upstream repository where I can duplicate these changes, so that they don't get overwritten in a future spec update?